canonical-websites/www.ubuntu.com
# add redirect /k8s and /kubernetes

## Done

* redirects for /k8s and /kubernetes to /containers/kubernetes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/kubernetes](http://0.0.0.0:8001/kubernetes) and [/k8s](http://0.0.0.0:8001/k8s) and that they go to  [/containers/kubernetes](http://0.0.0.0:8001/containers/kubernetes)
